sdk/eventhub/azure-eventhubs/azure/eventhub/common.py

```diff
@@ -260,6 +295,38 @@ def _set_partition_key(self, value):
             self.message.annotations = annotations
             self.message.header = header
 
+    def try_add(self, event_data):
+        """
+        The message size is a sum up of body, properties, header, etc.
+        :param event_data:
+        :return:
+        """
+        if event_data is None:
+            self.log.warning("event_data is None when calling EventDataBatch.try_add. Ignored")
```

Comment:
Should use module-level logger

Reply:
thanks for the suggestion, code updated.